src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,26 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
+			<wikit-button
+				class="form-button-submit"
+				type="progressive"
+				variant="primary"
+			>
+				{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
+			</wikit-button>
+		</div>
 	</form>
 </template>
 
 <style scoped lang="scss">
 @import "@wmde/wikit-tokens/variables";
 
 .new-lexeme-form {
+	// display
+	display: grid;
+	gap: $dimension-layout-xsmall;
```

Comment:
Wonderful! :rainbow:

Reply:
Wonderful indeed. Sadly, it didn't last 😔